srv/modules/runners/ready.py

```diff
@@ -75,11 +75,11 @@ def apparmor(self):
         for minion in contents:
             if contents[minion] and int(contents[minion]) == 0:
                 msg = "enabled on minion {}".format(minion)
-                if 'apparmor' in self.warnings:
-                    self.warnings['apparmor'].append(msg)
+                if 'apparmor' in self.passed:
+                    self.passed['apparmor'].append(msg)
```

Comment:
In testing this, I noticed that this change seems to have the consequence of potentially producing a very long validation string. The following is from a single-node cluster - I'm afraid that on a larger cluster the string might get overly long?

```
2018-03-26T13:45:53.547 INFO:teuthology.orchestra.run.target054038068139.stdout:Stage initialization output:
2018-03-26T13:45:53.547 INFO:teuthology.orchestra.run.target054038068139.stdout:firewall                 : [1m[92mdisabled[0m
2018-03-26T13:45:53.547 INFO:teuthology.orchestra.run.target054038068139.stdout:apparmor                 : [1m[92m['enabled on minion target054038068139.teuthology'][0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:DEV_ENV                  : [1m[92mTrue[0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:fsid                     : [1m[92mvalid[0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:public_network           : [1m[92mvalid[0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:cluster_network          : [1m[92mvalid[0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:cluster_interface        : [1m[92mvalid[0m
2018-03-26T13:45:53.548 INFO:teuthology.orchestra.run.target054038068139.stdout:monitors                 : [1m[92mvalid[0m
2018-03-26T13:45:53.549 INFO:teuthology.orchestra.run.target054038068139.stdout:mgrs                     : [1m[92mvalid[0m
2018-03-26T13:45:53.549 INFO:teuthology.orchestra.run.target054038068139.stdout:storage                  : [1m[92mvalid[0m
2018-03-26T13:45:53.549 INFO:teuthology.orchestra.run.target054038068139.stdout:ganesha                  : [1m[92mvalid[0m
2018-03-26T13:45:53.549 INFO:teuthology.orchestra.run.target054038068139.stdout:master_role              : [1m[92mvalid[0m
2018-03-26T13:45:53.549 INFO:teuthology.orchestra.run.target054038068139.stdout:time_server              : [1m[92mvalid[0m
2018-03-26T13:45:53.550 INFO:teuthology.orchestra.run.target054038068139.stdout:fqdn                     : [1m[92mvalid[0m
```